print/components/PicassoChunk.vue

```diff
@@ -240,13 +240,26 @@ export default {
 
 <style lang="scss">
 .landscape {
+  --font-scale: 0.67;
+  font-size: calc(10pt * var(--font-scale));
+
   transform: rotate(-90deg);
+  scale: 1.48;
```

Comment:
is the idea to have the same width as portrait, and then scale it up?

Reply:
Yes, correct, but it needs a bit of explanation.

I introduced a mistake in https://github.com/ecamp/ecamp3/commit/b8ddcd37e15bb773acfeba6088bc694bb9c65b98.
This commit upgraded chrome to 108, which [enabled the new LayoutNG for print](https://bugs.chromium.org/p/chromium/issues/detail?id=1121942#c175). When testing this upgrade, I noticed that all our content was scaled down. I couldn't find information, why this happened. I thought something changed on which DPI chrome uses for printing. My quick fix was to just [scale all content by factor 1.5](https://github.com/ecamp/ecamp3/commit/b8ddcd37e15bb773acfeba6088bc694bb9c65b98#diff-690f482f4da3d4ac6d4b33f0b145d860bad8cfb463b1161e403902f1d004e2f6R106). This was a mistake however, which can easily be seen by printing a single activity on devel (see also pdf included above).

What was really happening is the following:
- When the content includes elements that are wider than the page size, the new LayoutNG scales down all content in order to fit it on the page (instead of cropping the content)
- Our picasso landscape is theoretically wider than a A4 portrait page. However we rotate the picasso content, and evertthing should be fine. But it's not fine: the LayoutNG seems to check overflow content before applying the rotation, hence wrongly believing that we have too wide content and scaling down all content.

I will try to raise a bug with chrome, when I find some time.

The fix is to make the picasso width fit on a A4 portrait page, and then rotate + scale up in one transformation. This is a bit hacky, as the font size also needs to be scaled down in order to resemble more or less the same font size as before. But if works for now.

This can be reverted, once
- this specific bug is resolved in chrome
- or once chrome can properly handle mixed page orientation